<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UdpOpsWidget</class>
 <widget class="QWidget" name="UdpOpsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
    <property name="styleSheet">
     <string>font: 9pt "Segoe UI";
QPushButton{min-width:84px;padding:6px;border-radius:4px;background:#1976d2;color:white}
QPushButton:disabled{background:#9e9e9e}
QLineEdit,QTextEdit{font:9pt "Segoe UI"}</string>
    </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="bindLayout">
     <item>
      <widget class="QLineEdit" name="editBindPort">
       <property name="placeholderText">
        <string>bind port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnBind">
       <property name="text">
        <string>Bind</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="teRecv"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="sendLayout">
     <item>
      <widget class="QLineEdit" name="editPeer">
       <property name="placeholderText">
        <string>host:port</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="editSend"/>
     </item>
     <item>
      <widget class="QPushButton" name="btnSend">
       <property name="text">
        <string>Send</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="actionLayout">
     <item>
      <widget class="QPushButton" name="btnSaveAction">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
 <item>
  <widget class="QLabel" name="labelLoadedAction">
   <property name="text"><string></string></property>
  </widget>
 </item>
  <item>
   <widget class="QPushButton" name="btnClear">
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
  </item>
     <item>
      <widget class="QPushButton" name="btnLoadAction">
       <property name="text">
        <string>Load</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
